src/Sulu/Component/Content/Import/WebspaceImportInterface.php

```diff
@@ -30,7 +30,7 @@ interface WebspaceImportInterface
      * @param bool $overrideSettings
      * @param string $exportSuluVersion
      *
-     * @return array
+     * @return \stdClass
```

Comment:
are you sure that this change is correct?

Reply:
yes